packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -2,98 +2,65 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import AbstractModal from 'terra-abstract-modal';
 import FocusTrap from 'focus-trap-react';
-import * as KeyCode from 'keycode-js';
-import Button, { ButtonVariants } from 'terra-button';
+import Button from 'terra-button';
 import classNames from 'classnames';
 import classNamesBind from 'classnames/bind';
-import ThemeContext from 'terra-theme-context';
 import { FormattedMessage } from 'react-intl';
+import ThemeContext from 'terra-theme-context';
+
+import NotificationIcon from './_NotificationIcon';
+import ContentLayoutAsList from './_ContentLayoutAsList';
+
 import styles from './NotificationDialog.module.scss';
 
 const cx = classNamesBind.bind(styles);
 
-const variants = {
-  ALERT: 'alert',
-  ERROR: 'error',
-  WARNING: 'warning',
-  INFO: 'info',
-  SUCCESS: 'success',
-  CUSTOM: 'custom',
-};
+const variants = [
+  'hazard-high',
+  'hazard-medium',
+  'hazard-low',
+  'error',
+  'custom',
+];
 
 const propTypes = {
   /**
-   * Header of the notification-dialog.
-   */
-  header: PropTypes.string,
-  /**
-   * Title of the notification-dialog.
+   * The variant of notification to be rendered. This renders the dialog with the corresponding header and icon to the
+   * variant concept.
+   * Use one of `hazard-high`, `hazard-medium`, `hazard-low`, `error`, or `custom`.
```

Comment:
You could use use the generated enum value instead of repeating the variants here? either way.
![Screen Shot 2020-08-18 at 2 05 32 PM](https://user-images.githubusercontent.com/1588747/90554757-efd1d080-e15b-11ea-8a30-8b1a9249140c.png)

Reply:
removed: https://github.com/cerner/terra-framework/pull/1105/commits/c45612d9b69bb2982f5ca4adbcad9abaf6a161f5#diff-cb62dc7ce0197bc569a9952450bb9fe5L30